algorithms/unit_tests/TestStdAlgorithmsTransformInclusiveScan.cpp

```diff
@@ -295,15 +324,23 @@ void run_all_scenarios() {
     run_single_scenario<Tag, ValueType>(it, bop_t(), uop_t(), ValueType{2});
     run_single_scenario<Tag, ValueType>(it, bop_t(), uop_t(), ValueType{-1});
     run_single_scenario<Tag, ValueType>(it, bop_t(), uop_t(), ValueType{-2});
+
+    run_single_scenario_inplace<Tag, ValueType>(it, bop_t(), uop_t());
+    run_single_scenario_inplace<Tag, ValueType>(it, bop_t(), uop_t(),
+                                                ValueType{0});
+    run_single_scenario_inplace<Tag, ValueType>(it, bop_t(), uop_t(),
+                                                ValueType{2});
+    run_single_scenario_inplace<Tag, ValueType>(it, bop_t(), uop_t(),
+                                                ValueType{-2});
   }
 }
 
 #if !defined KOKKOS_ENABLE_OPENMPTARGET
 TEST(std_algorithms_numeric_ops_test, transform_inclusive_scan) {
   run_all_scenarios<DynamicTag, double>();
-  // run_all_scenarios<StridedThreeTag, double>();
-  // run_all_scenarios<DynamicTag, int>();
-  // run_all_scenarios<StridedThreeTag, int>();
```

Comment:
Why were these commented before?

Reply:
I think it was a leftover mistake from testing it manually